pkg/webui/console/views/application-collaborator-edit/index.js

```diff
@@ -0,0 +1,262 @@
+
+    dispatch(getApplicationCollaboratorPageData(appId))
+  }
+
+  async handleSubmit (values, { resetForm }) {
+    const { collaborator_id, rights } = values
+    const { appId, collaborator } = this.props
+    const collaborator_type = collaborator.isUser ? 'user' : 'organization'
+
+    const collaborator_ids = {
+      [`${collaborator_type}_ids`]: {
+        [`${collaborator_type}_id`]: collaborator_id,
+      },
+    }
+    const updatedCollaborator = {
+      ids: collaborator_ids,
+      rights: Object.keys(rights).filter(r => rights[r]),
+    }
+
+    await this.setState({ error: '' })
+
+    try {
+      await api.application.collaborators.update(appId, updatedCollaborator)
+      resetForm(values)
+      toast({
+        message: m.updateSuccess,
+        type: toast.types.SUCCESS,
+      })
+    } catch (error) {
+      resetForm(values)
+      await this.setState({ error })
+    }
+  }
+
+  async handleDelete () {
+    const { collaborator, appId, dispatch } = this.props
+    const collaborator_type = collaborator.isUser ? 'user' : 'organization'
+
+    const collaborator_ids = {
+      [`${collaborator_type}_ids`]: {
+        [`${collaborator_type}_id`]: collaborator.id,
+      },
+    }
+    const updatedCollaborator = {
+      ids: collaborator_ids,
+    }
+
+    try {
+      await api.application.collaborators.remove(appId, updatedCollaborator)
+      toast({
+        message: m.deleteSuccess,
+        type: toast.types.SUCCESS,
+      })
+      dispatch(replace(`/console/applications/${appId}/collaborators`))
+    } catch (error) {
+      await this.setState({ error })
+    }
+  }
+
+  render () {
+    const { collaborator, rights, fetching, error } = this.props
+
+    if (error) {
+      return 'ERROR'
+    }
+
+    if (fetching || !collaborator) {
+      return <Spinner center />
+    }
+
+    const hasUniversalRights = UNIVERSAL_APPLICATION_RIGHTS.reduce(
+      (acc, curr) => acc || collaborator.rights.includes(curr), false)
+    const { rightsItems, rightsValues } = rights.reduce(
+      function (acc, right) {
+        acc.rightsItems.push(
+          <Field
+            className={style.rightLabel}
+            key={right}
+            name={right}
+            type="checkbox"
+            title={{ id: `enum:${right}` }}
+            form
+          />
+        )
+        acc.rightsValues[right] = hasUniversalRights || collaborator.rights.includes(right)
+
+        return acc
+      },
+      {
+        rightsItems: [],
+        rightsValues: {},
+      }
+    )
+
+    const initialFormValues = {
+      collaborator_id: collaborator.id,
+      rights: { ...rightsValues },
+    }
+
+    return (
+      <Container>
+        <Row>
+          <Col lg={8} md={12}>
+            <IntlHelmet
+              title={sharedMessages.collaboratorEdit}
+              values={{ collaboratorId: collaborator.id }}
+            />
+            <Message component="h2" content={sharedMessages.edit} />
```

Comment:
```suggestion
            <Message component="h2" content={sharedMessages.collaboratorEdit} values={{ collaboratorId: collaborator.id }} />
```
Also, I believe the message would be better as: `Edit rights of {user}`